.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+<!-- Before submitting a PR, please ensure you have read the Arcade Contributor Guidance doc: https://github.com/dotnet/arcade/blob/master/Documentation/Policy/ArcadeContributorGuidance.md -->
```

Comment:
This doc doesn't include the coding guidelines or the other things that were in the checklist before :(

Reply:
will be adding!

Reply:
what does this xml comment looking thing do in the markdown? Does it get hidden in the actual PR?

Reply:
It doesn't show up in the markdown, but it will show up in the text box when you create the PR.

Reply:
I suggest having the text be more active. Instead of a comment that says "Please read this", have a checkbox that says "I have read this".

Reply:
I don't like checkboxes for something that doesn't change. I read the contribution guide once, but I'm not going to read it on every PR I make to this repo.

requesting actions from contributors like filling out checkboxes or sections of a template seem valuable for things that change from PR to PR, such as the validation method used, or the issue that your PR is addressing, but I don't think they are that useful for a static thing such as "are you sure you read the contributing doc", which is just going to get clicked on anyways.